### PR TITLE
chore: enable source maps

### PR DESCRIPTION
**Pull request by reidrankin**
## Description

Re-enable source maps so we can debug things in prod.

## Notice

Before submitting a pull request, please make sure you have answered the following:

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?
- [x] Do all new and existing tests pass? Does the linter pass?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

### CODE REVIEW

**Review by 0xdef1cafe — APPROVED** [review]

**Comment by 0xdef1cafe** [comment]
approved after discussion, please investigate flakey cloudflare builds before merging

**Review by 0xdef1cafe — APPROVED** [review]

**Review by cjthompson — APPROVED** [review]

**Pull request merged**
